pytest.ini

```diff
@@ -0,0 +1,6 @@
+[pytest]
+env =
+    FLASK_ENV=unit_test
+    FLASK_DEBUG=1
+    D:DATABASE_URL=postgresql://postgres:postgres@127.0.0.1:5432/postgres
```

Comment:
Looks like this `D:` is a typo?
```suggestion
    DATABASE_URL=postgresql://postgres:postgres@127.0.0.1:5432/postgres
```

Reply:
This means that if a environment variable is set called "DATABASE_URL" it will override the one set here, but maybe that's poor form?

Reply:
Isn't this already overridden in the unit_test config?

Reply:
Ah, OK, I assumed it was a typo but now see it's a pytest-env feature, happy to leave this, although as @harryyo says not sure how this works with the setting in the unit_test config?